include/af/statistics.h

```diff
@@ -232,6 +247,80 @@ AFAPI void topk(array &values, array &indices, const array& in, const int k,
                 const int dim = -1, const topkFunction order = AF_TOPK_MAX);
 #endif
 
+#if AF_API_VERSION >= 38
```

Comment:
It would be nice if we moved these definitions closer to the deprecated versions.

Reply:
Sure.